content/learn/implementation.adoc

```diff
@@ -2,73 +2,82 @@
 menu:
   learn:
     parent: Workflow
-title: Implementation Requirements
-weight: 41
+title: Implementation requirements
+weight: 42
 aliases: /requirements/implementation/
 ---
 
 :toc:
 
+:_content-type: ASSEMBLY
+include::modules/comm-attributes.adoc[]
+
 [id="technical-requirements"]
-== Technical Requirements
+== Technical requirements
+
+Consider these requirements specific to the implementation of all {solution-name-upstream} and their respective tiers.
 
-Additional requirements specific to the implementation for all Community, and Validated patterns
+The requirements are categorized as follows:
 
-[id="must"]
+Must::
+These are non-negotiable, core requirements that must be implemented.
+Should::
+These are important but not critical; their implementation enhances the pattern.
+Can::
+These are optional or desirable features, but their absence does not hinder the implementation of a  pattern.
```

Comment:
Additional space.

```suggestion
These are optional or desirable features, but their absence does not hinder the implementation of a pattern.
```

Also this list is a good definition of the requirement categories. Should we include these at the beginning of the pages for the other tiers as well?